components/public-api/typescript-common/src/public-api-converter.ts

```diff
@@ -880,6 +882,24 @@ export class PublicAPIConverter {
         });
     }
 
+    fromSort(sort: Sort) {
```

Comment:
JP wrote a sort util, you can re-use it or bring it here

https://github.com/gitpod-io/gitpod/blob/ft/global-prebuild-list/components/server/src/api/installation-service-api.ts#L55

Reply:
JP's util makes many assumptions. Did not want to refactor his code so I think having one which does not have any defaults inside it could be useful.